compiler/rustc_feature/src/active.rs

```diff
@@ -221,6 +221,8 @@ declare_features! (
     (active, rustc_private, "1.0.0", Some(27812), None),
     /// Allows using internal rustdoc features like `doc(primitive)` or `doc(keyword)`.
     (active, rustdoc_internals, "1.58.0", Some(90418), None),
+    /// Allows using the `rustdoc::missing_doc_code_examples` lint
+    (active, rustdoc_missing_doc_code_examples, "1.31.0", Some(101730), None),
```

Comment:
This is the version the original lint merged into, I'm not sure if this should just use the current nightly version instead though?

Reply:
I suppose it doesn't matter much. The important version is the stable one once it gets stabilized.